pkg/reconciler/integration/sink/integrationsink.go

```diff
@@ -45,10 +45,9 @@ import (
 	"knative.dev/eventing/pkg/auth"
 	eventingv1alpha1listers "knative.dev/eventing/pkg/client/listers/eventing/v1alpha1"
 
-	certmanagerclientset "knative.dev/eventing/pkg/client/certmanager/clientset/versioned"
```

Comment:
@skonto  these changes you did too, for serving right?

Reply:
Yes removed the dep on our generated clientset.